jaxdecomp/_src/fft.py

```diff
@@ -231,14 +231,16 @@ def lower_fn(operand):
 
     output = sfft(operand, fft_type, adjoint, pdims, global_shape)
 
-    # In case of YZ slab the cuda code tranposes only once
-    # We transpose again to give back the Z-Pencil to the user in case of FFT and the X-Pencil in case of IFFT
-    # this transposition is supposed to compiled out by XLA when doing a gradient (forward followed by backward)
-    if get_axis_size(input_sharding, 0) == 1:
-      if fft_type == FftType.FFT:
-        output = output.transpose((1, 2, 0))
-      elif fft_type == FftType.IFFT:
-        output = output.transpose((2, 0, 1))
+    # This is supposed to let us avoid making an extra transpose in the YZ case
```

Comment:
This did not work as expected
Tranposing in JAX is not equivalent to transposing in cuDecomp
Not in this stage at least (in here we are running code per shard basis, so transposes are local)